### ISSUE TITLE
Flaws

### ISSUE THREAD

**Issue by War-gamer** [dylan/gameofcyvasse#9]
My good ser,

I have playtested your game. As one who develops war games myself I would like to give you a few tips. First off, elephants and rabble are equal. Secondly the fact that combat is simply "attacker wins" light horse is superior to heavy horse. My third and final comment is that nothing stops the dragon from just killing the king immediately, or if he is garrisoned in a fortress, killing the enemy dragon.

**Comment by dylan** [comment]
Thank for the comments, a couple things: You realize there is a rank system, for example: the rabble can only beat other rabble, the king, and spearman?

Re: Horses, Light horse can move farther, but cannot kill elephants. So while more mobile, they aren't 'heavy' and don't have the rank.

Re: Dragons, I just realized that I never defined the fact that I imagined them moving like a queen in chess, ie: only straights and diagonals. Thanks for pointing that out!
